spec/system/login_spec.rb

```diff
@@ -83,15 +83,24 @@ def wait_for_email_link(user, type)
       user.update!(password:)
       Fabricate(:expired_user_password, user:, password:)
```

Comment:
what's the point of these empty keys here?

Reply:
It's just the shorthand of `Fabricate(:expired_user_password, user: user, password: password)`